src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -42,12 +42,14 @@ public void stopIntake(){
   }
 
   public void toggleIntake(){
-    // switch(intakeSolenoid.get()){
-    //     case kReverse:
-    //     setIntake(DoubleSolenoid.Value.kForward);
-    //     case kForward:
-    //     setIntake(DoubleSolenoid.Value.kReverse);
-   // }
+  //   switch(intakeSolenoid.get()){
+  //       case kReverse:
+  //       setIntake(DoubleSolenoid.Value.kForward);
+  //       case kForward:
+  //       setIntake(DoubleSolenoid.Value.kReverse);
+  //       default:
+  //       setIntake(DoubleSolenoid.Value.kOff);
```

Comment:
Move the default up to kForward. So 
caseKForward:
setIntake(DoubleSolenoid.Value.kReverse);
That way it will always toggle the cylinder